apps/ima/index.js

```diff
@@ -33,9 +36,65 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
+      next: '/human-rights-claim', // TO BE UPDATED AS STEPS ARE ADDED
       backLink: 'current-progress'
     },
+    '/human-rights-claim': {
+      behaviours: SaveFormSession,
+      fields: ['human-rights-claim'],
+      locals: { showSaveAndExit: true },
+      forks: [
+        {
+          target: '/human-rights-family',
+          condition: req => req.sessionModel.get('human-rights-claim') === 'yes',
+          continueOnEdit: true
+        },
+        {
+          target: '/other-human-rights-claims',
+          condition: req => req.sessionModel.get('human-rights-claim') === 'no',
+          continueOnEdit: true
+        }
+      ],
+      continueOnEdit: true,
+      next: '/other-human-rights-claims'
+    },
```

Comment:
I think
` '/human-rights-claim': {
      behaviours: SaveFormSession,
      fields: ['human-rights-claim'],
      locals: { showSaveAndExit: true },
      forks: [
        {
          target: '/human-rights-family',
          condition: {
          field: human-rights-claim',
          value: 'yes'
        }
      ],
      continueOnEdit: true,
      next: '/other-human-rights-claims'
    },`

is cleaner

Reply:
Changed the syntax of the forking conditions as suggested.